system/Test/FeatureResponse.php

```diff
@@ -40,6 +40,7 @@
 
 use CodeIgniter\HTTP\RedirectResponse;
 use CodeIgniter\HTTP\Response;
```

Comment:
should no longer needed as no longer used

Reply:
Fix it